x-pack/plugins/actions/server/lib/action_executor.test.ts

```diff
@@ -53,6 +54,7 @@ test('successfully executes', async () => {
   const actionType = {
     id: 'test',
     name: 'Test',
+    minimumLicenseRequired: 'basic' as LicenseType,
```

Comment:
seems like maybe we could avoid the `as` if `actionType` was typed explicitly to `ActionType`?  Not sure ...

Reply:
Oh, that's all it took! Thank you, fixed in https://github.com/elastic/kibana/pull/58668/commits/c87435b7c501adeec89f9634ccd0b781be979210.